src/operator/tensor/matrix_op-inl.h

```diff
@@ -674,13 +675,23 @@ inline void SetSliceOpOutputDimSize(const index_t i, const int b,
                                     const int e, const int s,
                                     TShape* oshape) {
   if (s > 0) {
-    CHECK_LT(b, e) << "slicing with begin=[" << i << "]=" << b << ", end[" << i << "]="
+    CHECK_LE(b, e) << "slicing with begin=[" << i << "]=" << b << ", end[" << i << "]="
```

Comment:
Is the case `b = e` required?

Reply:
Yes, now when shape of a dim is not known, `b==e==0`